kubernetes/otel-collector/otel-collector-config.yml

```diff
@@ -0,0 +1,70 @@
+receivers:
+  opencensus:
+    endpoint: 0.0.0.0:55678
+
+processors:
+  probabilistic_sampler:
+    sampling_percentage: 10
```

Comment:
is this default sampler we should go with? is this working considering we will be running service which are backed by multiple pods? and also can you please find out what is the default sampler we should go with?